config/routes.rb

```diff
@@ -1,3 +1,24 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, path: 'accounts', controllers: {
+    registrations: 'users/registrations'
+  }
+
+  root 'newsfeed#show'
+  
+  concern :likable do
```

Comment:
Please enlighten me on this block of code here :) I think we can simply do this in the `posts` resources below:

```ruby
resources :likes, module: :posts, only: %i[create destroy index]
```
To make it simpler. If you're wondering why I used `modules` here -> See this [tutorial](https://gorails.com/episodes/comments-with-polymorphic-associations) to make your polymorphic associations much easier to reason about

Reply:
@rvvergara, thanks for the suggestion. 

We are using concern to write a DRYer code. if we were to use `module:`, there would be a lot of duplicates and our code may even require us to nest the route more than one level deep.